.circleci/config.yml

```diff
@@ -78,6 +79,8 @@ installtorchgpu14: &installtorchgpu14
       pip3 install --progress-bar off 'torchtext==0.5.0'
       pip3 install --progress-bar off 'torchvision==0.5.0'
       pip3 install --progress-bar off transformers
+      pip3 install --progress-bar off snips-nlu
```

Comment:
what is snips-nlu used for? Does tokenizers *really* require this?

Reply:
This fixes a rust not found issue.

Reply:
I don't think we need it, can we try one more time to get rid of it with the requirements.txt change?